components/link/index.css

```diff
@@ -45,11 +46,14 @@ governing permissions and limitations under the License.
   }
 }
 
-.spectrum-Link--subtle,
-/** @deprecated */.spectrum-Link--quiet {
+.spectrum-Link--secondary {
```

Comment:
This needs to have `color: inherit` or something -- secondary links are supposed to look like the paragraph text. Not sure what to do on hover? https://spectrum.adobe.com/page/link/#Secondary-variant